src/main/java/com/redislabs/redisgraph/RedisGraphPipeline.java

```diff
@@ -57,17 +57,6 @@
      */
     Response<ResultSet> readOnlyQuery(String graphId, String query, long timeout);
 
-    /**
-     * Execute a Cypher query with arguments
-     * @param graphId a graph to perform the query on
-     * @param query Cypher query
-     * @param args
-     * @return a response which builds the result set with the query answer.
-     * @deprecated use {@link #query(String, String, Map)} instead.
-     */
-    @Deprecated
-    Response<ResultSet> query(String graphId, String query, Object ...args);
```

Comment:
Removing this means we'll have to target this PR for next major release. I don't think next major release is near.

Reply:
I shouldn't have add it from the beginning it was a mistake to add it to this new class when I copied it from the Transaction class.